### PR TITLE
feat: add translation web package

### PR DESCRIPTION
**Pull request by davidsoderberg**
### What change does this PR introduce?

Add translations routes

Screenshot 2023-10-24 at 13 26 59

### CODE REVIEW

File: apps/web/src/pages/TranslationPages.tsx
Comment (djabarovgeorge) [inline apps/web/src/pages/TranslationPages.tsx:1]: what do you think about importing only if NOVU_MANAGE is true, and if the import fails then throw an exception?
At the moment we could have the cloud without the translation page and we won't know about it.

File: libs/design-system/src/layout/PageContainer.tsx
Comment (djabarovgeorge) [inline libs/design-system/src/layout/PageContainer.tsx:1]: 🚀

File: apps/web/src/components/layout/components/PageContainer.tsx
[the remark this answers is not in the document]
Comment (davidsoderberg) [reply apps/web/src/components/layout/components/PageContainer.tsx:1]: We could I just did not like to make to huge changes in this PR keep it easy to review.

**Comment by djabarovgeorge** [comment]
Looks great but won't the installation fail because of the existence of non-existent on @novu/ee-translation-web in this state?

**Review by LetItRock — APPROVED** [review]
🚀

**Pull request merged**